tldextract/cache.py

```diff
@@ -219,7 +219,8 @@ def _fetch_url(session, url, timeout):
 def _make_cache_key(inputs):
     key = repr(inputs)
     try:
```

Comment:
@brycedrennan do you remember what this `try` was for? According to the type checker, `repr` above returns a `str`, so `md5(key)` will always raise.

Reply:
It's a good question. I'm not sure. the only thing I can think of is Python 2 compatibility.

Reply:
yeah I see you removed python 2 compat, so that's likely what this was for

Reply:
Thanks for confirming! Cool. Consider the `try` gone.